test/command/latex-multicols.md

```diff
@@ -0,0 +1,14 @@
+# `\begin{multicols}`
+
+```
+% pandoc -f latex -t native
+\begin{multicols}{2}
+Hello
+\end{multicols}
+^D
+[Div ("",["columns"],[])
+ [Div ("",["column"],[])
+  [Div ("",["column"],[])
+   [Para [Str "Hello"]]]]]
+```
```

Comment:
I think you may have misunderstood what I meant about two `column` Divs: the structure should be:
```
Div columns
[ Div column
, Div column ]
```
and not
```
Div columns
[ Div column
  [ Div column ...
```
Does that make more sense?

Reply:
Even simpler :) I will update

Reply:
Oh, what would be the content of the first div for example? Empty?

```
Div columns
[ Div column ""
, Div column "text foo hello"]
```
?

Reply:
Maybe I need to understand better what you're trying to do.

The latex multicols environment creates columned text, but it computes where the column breaks go.
In order to duplicate this in pandoc we'd need pandoc to compute where a new column starts.
Is this something you're trying to do?

Reply:
No, I am basically trying to ignore the number of columns in latex :)

```latex
\begin{multicols}{2} % <- here
% ...
\end{multicols}
```

There is nothing more we can do, I think.

Reply:
OK, then maybe I led you astray in my advice to include the nested column divs.
Maybe it makes most sense simply to include an outer "columns" div, and then the content inside.